### PR TITLE
Fix OfficeConnector CSS IDs in templates

### LINKED ISSUE #2581: Newstyle Officeconnector URLs are broken on tooltips and bumblebee overlays
I forgot to update the tooltip and bumblebee overlay templates for the new JS side changes inflicted in the pull request https://github.com/4teamwork/opengever.core/pull/2496.

### PR DESCRIPTION
**Pull request by Rotonen**
* Add tests for template integrity for the various similar subviews.
* Fix the subviews.

Fixes #2581.

### CODE REVIEW

File: opengever/officeconnector/tests/test_templates.py [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+                               .within(self.dossier)
+                               .attach_file_containing(
+                                   bumblebee_asset('example.docx')
+                                   .bytes(),
+                                   u'example.docx',
+                                   )
+                               )
+
+    def tearDown(self):
+        super(TestOfficeConnectorTemplates, self).tearDown()
```
Comment (deiferni) [inline opengever/officeconnector/tests/test_templates.py]: unneeded method, if it just calls `super`, drop it.

File: opengever/officeconnector/tests/test_templates.py [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+    @browsing
+    def test_tooltip_template_without_officeconnector(self, browser):
+        browser.login().open(self.document, view='tooltip')
+
+        self.assertEqual(0, len(browser.css('#officeconnector-attach-url')))
+        self.assertEqual(0, len(browser.css('#officeconnector-checkout-url')))
+
+    @browsing
+    def test_overview_template_with_officeconnector(self, browser):
+        browser.login().open(self.document, view='tabbedview_view-overview')
```
Comment (deiferni) [inline opengever/officeconnector/tests/test_templates.py]: i'd split this method into 3 separate ones that each test their single permutation value

File: opengever/officeconnector/tests/test_templates.py [hunk 3]
```diff
@@ -0,0 +1,257 @@
...
+            in browser.css('#officeconnector-attach-url')[0].outerHTML)
+
+        self.assertEqual(1, len(browser.css('#officeconnector-checkout-url')))
+        self.assertTrue(
+            'data-officeconnector-checkout-token-url'
+            in browser.css('#officeconnector-checkout-url')[0].outerHTML)
+
+    @browsing
+    def test_tooltip_template_with_officeconnector(self, browser):
+        browser.login().open(self.document, view='tooltip')
```
Comment (deiferni) [inline opengever/officeconnector/tests/test_templates.py]: also, split please

File: opengever/officeconnector/tests/test_templates.py [hunk 4]
```diff
@@ -0,0 +1,257 @@
...
+                               .within(self.dossier)
+                               .attach_file_containing(
+                                   bumblebee_asset('example.docx')
+                                   .bytes(),
+                                   u'example.docx',
+                                   )
+                               )
+
+    def tearDown(self):
+        super(TestOfficeConnectorBumblebeeTemplates, self).tearDown()
```
Comment (deiferni) [inline opengever/officeconnector/tests/test_templates.py]: drop method

File: opengever/officeconnector/tests/test_templates.py [hunk 5]
```diff
@@ -0,0 +1,257 @@
...
+    @browsing
+    def test_bumblebeeoverlay_template_without_officeconnector(self, browser):
+        browser.login().open(self.document, view='bumblebee-overlay-listing')
+
+        self.assertEqual(0, len(browser.css('#officeconnector-attach-url')))
+        self.assertEqual(0, len(browser.css('#officeconnector-checkout-url')))
+
+    @browsing
+    def test_bumblebeeoverlay_template_with_officeconnector(self, browser):
+        browser.login().open(self.document, view='tabbedview_view-overview')
```
Comment (deiferni) [inline opengever/officeconnector/tests/test_templates.py]: split 😉

**Comment by Rotonen** [comment]
@deiferni ready for rereview.

**Comment by Rotonen** [comment]
@lukasgraf rebased.

**Pull request merged**